### PR TITLE
fix #2994 InstructionListView fade edge color

### LINKED ISSUE #2994: InstructionListView's fading edge color is not matching
### Steps to trigger behavior

 1. Launch `example` app
 2. Launch `NavigationView` in `Navigation UI SDK` examples
 3. Click the top banner instruction view to show the `InstructionListView`
 4. Drag the `InstructionListView` down

### Actual behavior
The edge shows a purple like color. 
Screenshot_20200512-200326

### PR DESCRIPTION
**Pull request by JunDai**
## Description

Fix #2994

- [x] I have added any issue links
- [x] I have added all related labels (`bug`, `feature`, `new API(s)`, `SEMVER`, etc.)
- [ ] I have added the appropriate milestone and project boards

### Goal

Please describe the PR goals. Just the stuff needed to implement the fix / feature and a simple rationale. It could contain many check points if needed

### Implementation

The RecyclerView fading edge color is related to the current theme's `colorPrimary`. If you look at the other sample like: `InstructionViewActivity` and `CustomUIComponentStyleActivity`, they don't have this issue. So to assign a white color as `colorPrimary` for the default NavigationUISDK theme to have this behavior defined. The developer can still override this attribute when they extend from our theme.

## Screenshots or Gifs

Screenshot_20200518-170138_XRecorder

## Testing

Please describe the manual tests that you ran to verify your changes

- [x] I have tested locally (including `SNAPSHOT` upstream dependencies if needed) through testapp/demo app and run all activities to avoid regressions
- [x] I have tested via a test drive, or a simulation/mock location app
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have updated the `CHANGELOG` including this PR

### CODE REVIEW

File: libnavigation-ui/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -26,6 +26,7 @@
     </style>
 
     <style name="NavigationViewLight" parent="Theme.MaterialComponents.Light.NoActionBar">
+        <item name="android:colorPrimary" tools:targetApi="lollipop">@android:color/white</item>
```
Comment (Guardiola31337) [inline libnavigation-ui/src/main/res/values/styles.xml:29]: We support `minSdk 19` could targeting Lollipop only here be an issue?

File: libnavigation-ui/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -72,6 +73,7 @@
     </style>
 
     <style name="NavigationViewDark" parent="Theme.AppCompat.Light.NoActionBar">
+        <item name="android:colorPrimary" tools:targetApi="lollipop">@android:color/white</item>
```
Comment (Guardiola31337) [inline libnavigation-ui/src/main/res/values/styles.xml:76]: Same as above

File: libnavigation-ui/src/main/res/values/styles.xml (hunk 1 above)
Comment (JunDai) [reply libnavigation-ui/src/main/res/values/styles.xml:29]: no, verified on Android 19 emulator and it doesn't have side effect. and it works well.

Comment (Guardiola31337) [reply libnavigation-ui/src/main/res/values/styles.xml:29]: 🆗 Thanks for confirming. I've just noticed that this is `tools` 🤦 i.e. only for the IDE so I believe we can remove. Why do we need it 🤔?

**Pull request merged**